flink-connector-kudu/src/main/java/org/apache/flink/connectors/kudu/connector/writer/KuduWriterConfig.java

```diff
@@ -65,6 +101,11 @@ public String toString() {
     public static class Builder {
         private String masters;
         private FlushMode flushMode = FlushMode.AUTO_FLUSH_BACKGROUND;
+        private long timeout = 30000;
+        private int maxBufferSize = 1000;
+        private int flushInterval = 1000;
```

Comment:
Where did you take this defaults from? How will this affect the out of the box performance?

Reply:
These are from https://github.com/apache/kudu/blob/1eb2a2fbca/java/kudu-client/src/main/java/org/apache/kudu/client/AsyncKuduSession.java and  https://github.com/apache/kudu/blob/1eb2a2fbca/java/kudu-client/src/main/java/org/apache/kudu/client/AsyncKuduClient.java.  So, It will not affect the out of box performance.  I can add some  Annotation for them.

Reply:
sounds good